lib/index.js

```diff
@@ -17,6 +17,12 @@ module.exports = (logger) => {
             err = new Error(err);
         }
 
+        logger.error(`${req.originalMethod}: ${req.url}`);
+
+        if (Object.keys(req.body).length) {
```

Comment:
You should probably use `Object.getOwnPropertyNames()` to also handle non-enumerable properties.
```suggestion
        if (Object.getOwnPropertyNames(req.body).length) {
```

Reply:
Hi @FerrielMelarpis, thanks for commenting. I think this is fine if we don't know that req.body can have non-enumerable properties but since we are sure as of the express version that req.body won't have non-enumerable then Object.keys should be fine. https://expressjs.com/en/resources/middleware/body-parser.html